### ISSUE TITLE
libdnf5 plugins: vague error message while further dnf5 actions are blocked

### ISSUE THREAD

**Issue by glum23** [rpm-software-management/dnf5#1336]
When an incorrect (non-existent) plugin name is used in an /etc/dnf/libdnf5-plugins/ config file, dnf5 stops working and provides quite vague error message:
```
# dnf5 repo list
Cannot load plugins: actions.conf
# dnf5 install zsh
Cannot load plugins: actions.conf
# cat /etc/dnf/libdnf5-plugins/actions.conf
[main]
name = unknown
enabled = 1
```
It would be nice to 
1. provide more useful info like "Cannot load plugin 'unknown' enabled from /etc/dnf/libdnf5-plugins/actions.conf"
2. ignore the plugin instead of completely blocking any further dnf5 actions

In addition to that, it'd be nice to provide more docs related to dnf5 / libdnf5 plugins, their configs etc.

version: dnf5-5.1.15-20240321004900.23.ge0ae9be5.fc39

**Comment by ppisar** [comment]
I agree with the better error message, I don't agree with the ignorance. Some plugins can be critical. E.g. the not yet ported  DNF4 plugin which verifies PGP key in DNS.

**Comment by j-mracek** [comment]
This error cannot detect not ported DNF4 plugin, because they have a configuration file in a different path. Interesting thing is:
Missing configuration file => plugin is not loaded
Present configuration file, but plugin is not preset -> fail

In DNF4 - plugin configuration file was optional
In DNF4 - broken plugin triggers a warning

- This behavior was introduce after an incident where a third party plugin had broken DNF.

**Comment by ppisar** [comment]
I used the PGP plugin as an example of a plugin where a broken plugin should prevent DNF from continuing. I did not mean that DNF5 should attempt to load DNF4 plugins.
